scico/linop/optics.py

```diff
@@ -144,7 +144,7 @@ def __init__(
         self.k0: float = k0
         #: Shape of input after padding
         self.padded_shape: Shape = tuple(pad_factor * s for s in input_shape)
-        #: Padded source plane side length `(dx[i] * padded_shape[i])`
+        #: Padded source plane side length (dx[i] * padded_shape[i])
```

Comment:
Why the colons at the start of the comment?

Reply:
Apparently it's sphinx syntax for [documenting attributes](https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#directive-autoattribute).